src/lib/components/icons/FilterIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import { CURRENT_COLOR_CLASS_NAME, SvgIcon, ISvgIconProps } from './SvgIcon';
+
+export const FilterIcon = (props: ISvgIconProps) => {
```

Comment:
see my other comment about props. after modify the props you can use:
```export const FilterIcon = ({...svgProps,isActive}: IFilterIconProps) => {```
then use this `isActive` directly in your code